import_export/forms.py

```diff
@@ -151,6 +151,7 @@ def _create_boolean_fields(self, resource: ModelResource, index: int) -> None:
             field_name = self.create_boolean_field_name(resource, field)
             boolean_field = forms.BooleanField(
                 label=self._get_field_label(resource, field),
+                label_suffix="",
```

Comment:
just wondering, was there a reason this needed to be added?

Reply:
It's to remove the colon that Django automatically adds for all labels.

Let me know if any changes should have comments